draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -491,22 +494,22 @@ congestion control to avoid overloading the network.
 
 QUIC is a congestion controlled transport protocol. Senders are required to
 employ some form of congestion control. The default congestion control specified
-for QUIC is an alogrithm similar to TCP NewReno, but senders are free to choose
+for QUIC is an alogrithm similar to TCP NewReno {{RFC9002}}, but senders are free to choose
```

Comment:
```suggestion
for QUIC is an alogrithm similar to TCP NewReno {{!RFC9002}}, but senders are free to choose
```
`!` makes it a normative reference, `?` for informative.

Reply:
This reference and text is confusing. The text says TCP Reno but then it refers to QUIC Reno. I would just change this to, `The default congestion control specified for QUIC is Reno {{!RFC 9002}}`

Reply:
@goelvidhi is right - this text/reference **is** scrambled. My thoughts are

- TCP NewReno is [RFC6582]
- the default QUIC congestion controller is [RFC9002] 
- [RFC9002] says 
> This document specifies a sender-side congestion controller for QUIC similar to TCP NewReno [RFC6582].

So, putting all this together, I get 

> The default congestion control specified for QUIC in {{RFC9002}} is similar to TCP NewReno {{RFC6582}}

which (IIUC) gives the references for QUIC's default congestion controller and TCP NewReno correctly. 

Please let me know if this is a Really Bad Idea.

Reply:
Sounds good to me.